general/auth/google.rst

```diff
@@ -1,49 +1,95 @@
+#. Click on *Create Project* on the right, and enter the project name and other details of your 
+   company.
 
-.. image:: media/google04.png
-    :align: center
+#. Click on *Create*.
 
-- Fill in your address, email and the product name (for example odoo) and then save.
+   .. image:: media/google-auth-create-project.png
+      :align: center
+      :alt: Create project button
 
-.. image:: media/google05.png
-    :align: center
+   .. image:: media/google-auth-new-project-details.png
+      :align: center
+      :alt: Filling in the details of a new project
 
-- Then click on **Add Credentials** and select the second option (OAuth 2.0 Client ID).
+   .. note::
+      - Type the name of your own company, NOT odoo.com as in the example.
 
-.. image:: media/google06.png
-    :align: center
+#. On the left side menu, click on :menuselection:`OAuth consent screen`.
 
-.. image:: media/google07.png
-    :align: center
+   .. image:: media/google-oauth-consent-selection.png
+      :align: center
+      :alt: Google oauth consent selection menu
 
-- Check that the application type is set on **Web Application**. Now configure the allowed pages on which you will be redirected.
+#. Choose one of the options **(Internal / External)** as instructed, and click on *Create*.
 
-- To achieve this, complete the field **Authorized redirect URIs**. Copy paste the following link in the box: http://mydomain.odoo.com/auth_oauth/signin. Then click on **Create**
+   .. image:: media/google-oauth-consent.png
+      :align: center
+      :alt: Choice of a user type in oauth consent
 
-.. image:: media/google08.png
-    :align: center
+#. Fill in your details, then click on *Save and Continue*.
 
-.. image:: media/google09.png
-    :align: center
+   .. image:: media/google-auth-edit-app-registration.png
+      :align: center
+      :alt: Edit app registration page
 
-- Once done, you receive two information (your Client ID and Client Secret). You have to insert your Client ID in the **General Settings**.
+   .. image:: media/google-auth-edit-app-registration-part2.png
+      :align: center
+      :alt: Edit app registration page part 2
+
+#. Click on *Save and Continue* once again, then click on *Back to Dashboard*.
+
+#. On the left side menu, click on :menuselection:`Credentials`.
+
+   .. image:: media/google-auth-credentials-button.png
+      :align: center
+      :alt: Credentials button menu
+
+#. Click on *Create Credentials* and select **OAuth client ID**.
+
+   .. image:: media/google-oauth-client-id.png
+      :align: center
+      :alt: Oauth client id selection
+
+#. Select **Web Application** as the Application type. Now configure the allowed pages on which you 
+   will be redirected. To achieve this, complete the field **Authorized redirect URIs**. Copy paste 
+   the following link into the box: http://mydomain.odoo.com/auth_oauth/signin (replace "mydomain"
+   with your own domain). Then click on *Create*.
```

Comment:
This point is bulky and full of really important points.
If your doc is separated in paragraphs, it could be easier to present in a clearer way with a list, for example.

The URL is not well presented. Not all databases are formatted this way. Even for those that are, the link you provided won't work as there is a "s" missing in `https`.
I would write it this way 

> In the **Authorized redirect URIs** field, enter your database's domain immediately followed by `/auth_oauth/signin`. For example: `https://mydomain.odoo.com/auth_oauth/signin`

Also, consider using backticks around the URI , to make it look nicer 
\`https://mydomain.odoo.com/auth_oauth/signin` --> `https://mydomain.odoo.com/auth_oauth/signin`